modules/migration-excluding-resources.adoc

```diff
@@ -41,10 +43,12 @@ spec:
   - serviceclasses
   - serviceinstances
   - serviceplans
+  - operatorgroups
+  - events
 ----
 <1> Add `disable_image_migration: true` to exclude image streams from the migration. Do not edit the `excluded_resources` parameter. `imagestreams` is added to `excluded_resources` when the `MigrationController` pod restarts.
 <2> Add `disable_pv_migration: true` to exclude PVs from the migration plan. Do not edit the `excluded_resources` parameter. `persistentvolumes` and `persistentvolumeclaims` are added to `excluded_resources` when the `MigrationController` pod restarts. Disabling PV migration also disables PV discovery when you create the migration plan.
-<3> You can add {product-title} resources to the `excluded_resources` list. Do not delete any of the default excluded resources. These resources are known to be problematic for migration.
+<3> You can add {product-title} resources to the `excluded_resources` list. Default excluded resources are shown. These resources are known to be problematic for migration. Do not delete any of them.
```

Comment:
Why did you change active to passive voice in second sentence? What was the problem with the original?

Reply:
I moved the second sentence; I didn't change its voice. I added a new sentence to make it clear that the resources shown were the default resources, just in case the user might think they were samples. If you think that's obvious from the original, I'll get rid of the changes.

Reply:
Who wrote "are shown"? That is what I think should be rewritten.

Reply:
I think this is clearer:

You can add {product-title} resources to the `excluded_resources` list. Do not delete the default excluded resources. These resources are problematic to migrate and must be excluded.